lib/iris/tests/experimental/test_fieldsfile.py

```diff
@@ -1,65 +0,0 @@
-# (C) British Crown Copyright 2014 - 2016, Met Office
```

Comment:
I think it could be worthwhile to retain this as a totally-simple integration test.
`from iris.experimental.fieldsfile import load` --> `from iris.fileformats.um import load_cubes as load`
And then rename --> `iris.tests.integration.um.test_ff_load` or somesuch ?